bin/foreman-proxy-certs-generate

```diff
@@ -20,20 +20,26 @@ class ForemanProxyCertsGenerate
 
   private
 
+  # rubocop: disable Metrics/MethodLength
   def generate_answers_file
+    if File.file?('./config/last_scenario.yaml')
+      scenario = YAML.load_file('./config/last_scenario.yaml').fetch(:answer_file)
+      organization = YAML.load_file("#{scenario}")['foreman']['initial_organization'] rescue nil
```

Comment:
Actually sorry, this should probably be `rescue "Default Organization"`, if for some reason there's a problem reading the file, making it nil will cause a problem down on Line 37 later